fatiando/mesher.py

```diff
@@ -1721,3 +1721,594 @@ def keep(cell):
             return True
         return False
     return [c for c in cells if keep(c)]
+
+
+class TriaxialEllipsoid(GeometricElement):
+
+    """
+    Create an arbitrarily-oriented triaxial ellipsoid.
+
+    Triaxial ellipsoids are those having different semi-axes.
+    This code follows Clark et al. (1986) and defines the spatial
+    orientation of the ellipsoid by using three angles: strike,
+    rake and dip. These angles are commonly used to define the
+    orientation of geological structures (Allmendinger et al., 2012).
+
+    References:
+
+    Allmendinger, R., Cardozo, N., and Fisher, D. M.: Structural geology
+    algorithms : vectors and tensors, Cambridge University Press, 2012.
+
+    Clark, D., Saul, S., and Emerson, D.: Magnetic and gravity anomalies
+    of a triaxial ellipsoid, Exploration Geophysics, 17, 189-200, 1986.
+
+    .. note:: The coordinate system used is x -> North, y -> East
+    and z -> Down
+
+    Parameters:
+
+    * x, y, z : float
+        The coordinates of the center of the ellipsoid.
+    * large_axis, intermediate_axis, small_axis: float
+        Semi-axes forming the ellipsoid (in m).
+    * strike, dip, rake
+        Orientation angles of the ellipsoid (in degrees).
+    * props : dict
+        Physical properties assigned to the ellipsoid.
+        Ex: ``props={'density':10,
+                     'remanent magnetization':[10, 25, 40],
+                     'susceptibility tensor':[0.562, 0.485, 0.25,
```

Comment:
Is the `"remanent magnetization"` something that you'll need in the forward modeling later as well?